setup.cfg

```diff
@@ -57,7 +57,7 @@ numba_extensions =
     init = vector:register_numba
 
 [flake8]
-ignore = E203, E231, E501, E722, W503, B950
-select = C,E,F,W,T,B,B9
+extend-ignore = E203, E231, E501, E722, W503, B950, D103, D102, D101, D100, D107, D105, D205, D400, D401, D104, D412
```

Comment:
Some of the basic ones here can be removed. I've forgotten which ones - I'd try removing all the "old" ones and seeing which ones show up.

Reply:
Done! Thanks!